ql/src/java/org/apache/hadoop/hive/ql/lockmgr/DbTxnManager.java

```diff
@@ -495,6 +501,7 @@ public void replCommitTxn(CommitTxnRequest rqst) throws LockException {
         // For transaction started internally by repl load command, heartbeat needs to be stopped.
         clearLocksAndHB();
       }
+      rqst.setTxn_type(TxnType.REPL_CREATED);
```

Comment:
rqst object is getting created somewhere else, why not to set the type right there?